listenbrainz/webserver/static/js/src/personal-recommendations/PersonalRecommendationsModal.tsx

```diff
@@ -0,0 +1,258 @@
+      suggestions: [],
+    });
+    // update the followers list
+    APIService.getFollowersOfUser(currentUser.name)
+      .then((response) => {
+        this.setState({ followers: response.followers });
+      })
+      .catch((error) => {
+        this.handleError(error, "Error while fetching followers");
+      });
+  };
+
+  submitPersonalRecommendation = async () => {
+    const { recordingToPersonallyRecommend, newAlert } = this.props;
+    const { blurbContent, users } = this.state;
+    const { APIService, currentUser } = this.context;
+
+    if (recordingToPersonallyRecommend && currentUser?.auth_token) {
+      const metadata: UserTrackPersonalRecommendationMetadata = {
+        artist_name: getArtistName(recordingToPersonallyRecommend),
+        track_name: getTrackName(recordingToPersonallyRecommend),
+        release_name: recordingToPersonallyRecommend!.track_metadata
+          .release_name,
+        recording_mbid: getRecordingMBID(recordingToPersonallyRecommend),
+        recording_msid: getRecordingMSID(recordingToPersonallyRecommend),
+        users,
+        blurb_content: blurbContent,
+      };
+      try {
+        const status = await APIService.submitPersonalRecommendation(
+          currentUser.auth_token,
+          currentUser.name,
+          metadata
+        );
+        if (status === 200) {
+          newAlert(
+            "success",
+            `You recommended this track to ${users.length} user${
+              users.length > 1 ? "s" : ""
+            }`,
+            `${metadata.artist_name} - ${metadata.track_name}`
+          );
+          this.setState({ blurbContent: "" });
+        }
+      } catch (error) {
+        this.handleError(error, "Error while recommending a track");
+      }
+    }
+  };
+
+  render() {
+    const { recordingToPersonallyRecommend } = this.props;
+    if (!recordingToPersonallyRecommend) {
+      return null;
+    }
+    const { blurbContent, users, suggestions } = this.state;
+    const {
+      track_name,
+      artist_name,
+    } = recordingToPersonallyRecommend.track_metadata;
+    const { APIService, currentUser } = this.context;
+    return (
+      <div
+        className="modal fade"
+        id="PersonalRecommendationModal"
+        tabIndex={-1}
+        role="dialog"
+        aria-labelledby="PersonalRecommendationModalLabel"
+        data-backdrop="static"
+      >
+        <div className="modal-dialog" role="document">
+          <form className="modal-content">
+            <div className="modal-header">
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+                aria-label="Close"
+                onClick={this.closeModal}
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 className="modal-title" id="PersonalRecommendationModalLabel">
+                Personally recommend this recording
```

Comment:
I think what would be great here would be to use the recording name instead.
Maybe something like `Recommend <b>${track_name}</b> to a friend`, what do you think ?

![image](https://user-images.githubusercontent.com/6179856/196189205-870f243f-bf78-4ce0-b845-36909d2023e2.png)